pkgs/development/python-modules/anyqt/default.nix

```diff
@@ -0,0 +1,23 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+}:
+
+buildPythonPackage rec {
```

Comment:
Since those are all Python packages, please add an `pythonImportsCheck`.

Reply:
Done

Reply:
This one is missing.